src/mds/LocalLockC.h

```diff
@@ -18,9 +18,9 @@
 
 #include "SimpleLock.h"
 
-class LocalLock : public SimpleLock {
+class LocalLockC : public SimpleLock {
```

Comment:
Reason for this rename?

Reply:
We had a name collision with a Windows function: https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-locallock

Reply:
I've just realized that one commit that was fixing name collisions was squashed by mistake so I added it back.

Reply:
@batrick Do you have an alternative suggestion or are you OK w/ `LocalLockC`?

Reply:
It's okay for now. I don't want to block this PR on something so trivial. We'll end up namespacing the MDS eventually to avoid this in the future.